templates/_formhelpers.html

```diff
@@ -33,7 +33,7 @@
     {% endfor %}
 {%- endmacro %}
 
-{% macro render_select2_field(field, tooltip=None) -%}
+{% macro render_select2_field(field, tooltip=None, config={}) -%}
```

Comment:
Should the default value for config contain the original tooltip for backwards compatibility?

```
config={ placeholder: "Select multiple" }
```

Reply:
Yes, done.